### PR TITLE
Safe 286

### PR DESCRIPTION
**Pull request by adrianmujica153**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,12 +53,15 @@
     "@progress/kendo-drawing": "^1.5.8",
     "@progress/kendo-recurrence": "^1.0.0",
     "@progress/kendo-theme-default": "latest",
+    "@types/lodash-es": "^4.17.4",
```
Comment (AntoineRelief) [inline package.json]: ⚠️ @adrianmujica153 For imports of types, use the --save-dev flag with npm i, so it's put under devDependencies ( no need to have them for builds )

File: projects/back-office/src/app/dashboard/pages/applications/applications.component.scss [hunk 2]
```diff
@@ -23,3 +23,10 @@
     margin-left: 12px;
   }
 }
+
+:host {
```
Comment (AntoineRelief) [inline projects/back-office/src/app/dashboard/pages/applications/applications.component.scss]: ⚠️ @adrianmujica153 put at the top of file :host

File: projects/safe/src/lib/components/layout/layout.component.scss [hunk 3]
```diff
@@ -9,7 +9,6 @@
 }
 
 .sidenav-content {
-    min-height: calc(100% - 64px);
```
Comment (AntoineRelief) [inline projects/safe/src/lib/components/layout/layout.component.scss:12]: @adrianmujica153 is that needed for the MR ?

Comment (adrianmujica153) [reply projects/safe/src/lib/components/layout/layout.component.scss:12]: I think that is not necessary.

**Pull request closed without merging**